.github/workflows/python-cd.yml

```diff
@@ -80,6 +80,8 @@ jobs:
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
+        with:
+          version: v0.9.1
```

Comment:
there seems to be a new suggested approach: https://github.com/docker/setup-buildx-action/issues/187#issuecomment-1398199592